pages/common/dolt-commit.md

```diff
@@ -0,0 +1,32 @@
+# dolt commit
+
+> Commit staged changes to tables.
+> More information: <https://docs.dolthub.com/interfaces/cli#dolt-commit>.
+
+- Commit all staged changes, opening the editor specified by `$EDITOR` to enter the commit message:
+
+`dolt commit`
+
+- Commit all staged changes with the specified message:
+
+`dolt commit --message "{{commit_message}}"`
+
+- Stage all unstaged changes to tables before committing:
+
+`dolt commit --all`
+
+- Use the specified commit date (defaults to current date and time):
+
+`dolt commit --date "{{date}}"`
```

Comment:
Could you maybe give an example, so it is clear in which format the date has to be specified?

Reply:
It supports ISO 8601 dates. How about this? `2021-01-31T00:00:00`

Reply:
Sure, looks good.